### DISCUSSION TITLE
Not able to create a taxonomy term in PHP

### DISCUSSION THREAD

**Discussion by jarnalyrkar** [statamic/cms#7710] [Troubleshooting] [answered]
The documentation explains how to create a taxonomy term from PHP here: https://statamic.dev/repositories/term-repository#creating
However, I get an error on the save() method, that reads "Call to a member function all() on null".

My code is this:
```
$term = Term::make()->taxonomy('curators')->slug('term_' . $item->id)->blueprint('curator');
$term->dataForLocale('en_US', [
  'title' => $item->name,
  'api_id' => $item->id
]);
$term->save();
```

If i dump the object before saving, this is what it looks like:
![image](https://user-images.githubusercontent.com/20617365/225608177-1f885962-e91d-4f0a-aeeb-a34be1bdbf47.png)

Any suggestions?

**Comment by jasonvarga** [comment, marked as answer]
In the `dataForLocale` method, the locale argument should be your site handle.

**Reply by jarnalyrkar** [reply]
Thank you very much, @jasonvarga, I've got it working now! 
I also made a pull request to the docs to make it more clear what arguments the method is expecting :)
